examples/ampi/Cjacobi3D/Makefile

```diff
@@ -16,10 +21,15 @@ jacobi-cpp: jacobi-cpp.C
 	$(CHARMC) -c jacobi-cpp.C 
 	$(CHARMC) -o jacobi-cpp jacobi-cpp.o -module CommonLBs -lm
 
-jacobi: jacobi.C
+jacobi.o: jacobi.C
 	$(CHARMC) -c jacobi.C 
+
+jacobi: jacobi.o
 	$(CHARMC) -o jacobi jacobi.o -module CommonLBs -lm
```

Comment:
Might as well remove explicit linking with CommonLBs here and elsewhere

Reply:
Done